krun/platform.py

```diff
@@ -26,13 +26,11 @@
 class BasePlatform(object):
     __metaclass__ = ABCMeta
 
-    # Each temperature sensor must cool to its starting temperature plus
-    # TEMP_THRESHOLD_DEGREES. An absolute value in degrees centigrade.
-    TEMP_THRESHOLD_DEGREES = 1
+    TEMP_THRESHOLD_DEGREES = 3
 
     TEMP_MANDATORY_WAIT = 1
     TEMP_POLL_FREQ = 10                 # seconds between polls
-    TEMP_POLLS_BEFORE_MELTDOWN = 60     # times 10 = ten mins
+    TEMP_POLLS_BEFORE_MELTDOWN = 360    # times 10 = 1 hour
```

Comment:
Can we make this time in seconds, rather than forcing the user to (slightly weirdly) have to divide it by `TEMP_POLL_FREQ`?